{{cookiecutter.app_slug}}/tests/test_{{cookiecutter.app_module}}.py

```diff
@@ -31,4 +31,4 @@ def test_api_echo(client):
 def test_api_echo_empty(client):
     """Should return a 404 error"""
     resp = client.get('/api/echo')
-    assert resp.status_code == 404
\ No newline at end of file
+    assert resp.status_code == 404
```

Comment:
I'm not sure what this one is. I don't remember changing anything here.

Reply:
it's a new line that I'm reasonably certain your editor automatically added when you saved the file